### PR TITLE
openrazer: new, 3.3.0

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

openrazer: new, 3.3.0

Package(s) Affected
-------------------

openrazer: 3.3.0

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by eatradish** [comment]
LGTM on amd64:

image

File: extra-python/python-daemonize/spec [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VER="2.5.0"
+SRCS="tbl::https://pypi.org/packages/source/d/daemonize/daemonize-$VER.tar.gz"
+CHKSUMS="sha256::dd026e4ff8d22cb016ed2130bc738b7d4b1da597ef93c074d2adb9e4dea08bc3"
```
Comment (MingcongBai) [inline extra-python/python-daemonize/spec:3]: Missing CHKUPDATE.

File: extra-x11/razergenie/spec [hunk 2]
```diff
@@ -0,0 +1,3 @@
+VER="0.9.0"
+SRCS="tbl::https://github.com/z3ntu/RazerGenie/releases/download/v$VER/RazerGenie-$VER.tar.xz"
+CHKSUMS="sha256::e4a35ce56f7a8bc102afaca121668831dab876a6f487849bce46b0c6613aa85e"
```
Comment (MingcongBai) [inline extra-x11/razergenie/spec:3]: Missing CHKUPDATE.

File: extra-kernel/openrazer/spec [hunk 3]
```diff
@@ -0,0 +1,3 @@
+VER="3.3.0"
+SRCS="tbl::https://github.com/openrazer/openrazer/releases/download/v$VER/openrazer-$VER.tar.xz"
+CHKSUMS="sha256::cb7bc5c4761f601c1f0da36022153382384058f61fb72b23ee02c575ec41546a"
```
Comment (MingcongBai) [inline extra-kernel/openrazer/spec:3]: Missing CHKUPDATE.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**